### ISSUE TITLE
The question of manual correction facial points

### ISSUE THREAD

**Issue by Wyyghst** [AlfredXiangWu/face_verification_experiment#23]
I want to use modelB to reimplement LFW verification. I used 'code_face' and 'code_point' to generate LFW facial points. I set fixed position and size bboxes for all no detected bbox pictures. And I selected the largest area bbox for multiple bboxes pictures. The LFW validation accuracy is 96.05%.
Then I checked all LFW picture facial points, and fix 96 pictures facial points by hand. Now the LFW validation accuracy is 96.45%. But still below your results.
I used LFW data that deep_funneling downloaded from official website, And I only fixed facial points obvious error(facial points on wrong person or significant dislocation). Then I used 'face_db_align.m' to align images. In the end, I used matcaffe to extraction features, calculation cosine similarity, and using SVM  to classification.

My problem is that which incorrect step made low validation accuracy?

By the way, I used features in 'LightenedCNN_B_lfw.mat', then reduction dimension to 100 by PCA, and can get 98.30% accuracy. Why dimension reduction, but the correct rate improve?
Thanks.

**Comment by AlfredXiangWu** [comment]
To be honest, I have no idea about your low accuracy. Do you train the model using “crop_size”? I convert the WebFace image into 144*144 and use "crop_size:128" in data_layer. 

PCA can treated as a linear projection to convert the correlated features into an uncorrelated feature space. However, for the PCA method, I want to know how you obtain the projection matrix. If you only do PCA and get the matrix on LFW, I think you are incorrect. The projection matrix should be obtained by PCA for the training set(CASIA-WebFace).

**Comment by Wyyghst** [comment]
Hi, AlfredXiangWu
The matlab code like this:

``` matlab
load('LightenedCNN_B_lfw.mat');
dimension = 100;
[COEFF,SCORE] = princomp(features, 'econ');
features = SCORE(:, 1:dimension);
```

I only do PCA and get the matrix on LFW. Why this method incorrect?
Thanks.

**Comment by AlfredXiangWu** [comment]
For example, if you have only one image to extract feature, how do you obtain the projection matrix?

In my opinion, the PCA should be done for CASIA-WebFace to obtain the projection matrix, and then you should use the obtained matrix to project the 256-d lfw features into a new feature space and evaluate the results.
